File: algorithm/BruteForce/BOJ_1065.py
import sys
sys.stdin = open('../input.txt', 'r')


n = int(input())
if n <= 99:
    print(n)
else:
    ans = 99
    for num in range(111, n + 1):
        num = str(num)
        diff = int(num[0]) - int(num[1])
        for i in range(1, len(num) - 1):
            if int(num[i]) - int(num[i + 1]) != diff:
                break
        else:
            ans += 1
    print(ans)

File: javascript/algorithm/BruteForce/BOJ_1065.js
let fs = require('fs');
let input = fs.readFileSync('input.txt').toString();
let ans = 0;
for (let index = 1; index <= input; index++) {
    if (index <= 99) {
        ans++
    } else {
        let n = index.toString();
        const diff = n[0] - n[1]
        let find = true
        for (let i = 2; i < n.length; i++) {
            if (n[i-1] - n[i] !== diff) {
                find = false
                break
            }
        }
        if (find) {
            ans++
        }
    }
};
console.log(ans)